release-notes/unreleased/IGAPP-1076-show-description-of-events-in-overview.yml

```
@@ -0,0 +1,6 @@
+issue_key: IGAPP-1076
+show_in_stores: true
+platforms:
+  - native
```

Not a valid platform, should be
```suggestion
  - android
  - ios
```